portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Key;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that deletes data from the database. */
+@WebServlet("/delete-data")
```

Comment:
I personally am not a fan of this, but Servlets do have a doDelete method: https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServlet.html#doDelete-javax.servlet.http.HttpServletRequest-javax.servlet.http.HttpServletResponse-

In REST API for example methods are used like this:
doPost -  to create a new resource
doPut - to update a resource
doGet - to get a resource
doDelete - to delete a resource

If you are interested instead of defining a new servlet to delete you could simply override doDelete() method.

Reply:
That's interesting, I didn't know that, I'll keep that in mind for later.